docs/dev/table/common.md

```diff
@@ -743,9 +743,9 @@ TableEnvironment tableEnv = ...; // see "Create a TableEnvironment" section
 final Schema schema = new Schema()
     .field("a", DataTypes.INT())
     .field("b", DataTypes.STRING())
-    .field("c", DataTypes.LONG());
+    .field("c", DataTypes.BIGINT());
 
-tableEnv.connect(new FileSystem("/path/to/file"))
+tableEnv.connect(new FileSystem().path("/path/to/file"))
```

Comment:
This error exists in several places in the documentation. Could you also update them?

Reply:
> This error exists in several places in the documentation. Could you also update them?

@dianfu ok， It has been updated.